sdk/src/Services/DynamoDBv2/Custom/DocumentModel/_async/DocumentBatchWrite.Async.cs

```diff
@@ -66,7 +72,11 @@ public partial class MultiTableDocumentBatchWrite
         /// <returns>A Task that can be used to poll or wait for results, or both.</returns>
         public Task ExecuteAsync(CancellationToken cancellationToken = default(CancellationToken))
         {
-            return ExecuteHelperAsync(cancellationToken);
+            var operationName = DynamoDBTelemetry.ExtractOperationName(nameof(MultiTableDocumentBatchWrite), nameof(ExecuteAsync));
+            using (DynamoDBTelemetry.CreateSpan(TracerProvider, operationName, spanKind: SpanKind.CLIENT))
+            {
+                return ExecuteHelperAsync(cancellationToken);
```

Comment:
Do we need to switch this to use `async` / `await` to maintain trace context? I thought we had to do this for the other PRs. If so there are a few places like this that need to be fixed up.

Reply:
I made this change for most of the async APIs but missed a few spots. Fixed them.